docs/core/diagnostics/sos-debugging-extension.md

```diff
@@ -0,0 +1,369 @@
+
+## Remarks
+
+The SOS Debugging Extension lets you view information about code that is running inside the CLR. For example, you can use the SOS Debugging Extension to display information about the managed heap, look for heap corruptions, display internal data types used by the runtime, and view information about all managed code running inside the runtime.
+
+### Dotnet-Dump
+
+Please see [dotnet-dump](dotnet-dump.md#analyze-sos-commands) for list of available SOS commands with `dotnet-dump analyze`.
+
+### Windows Debugger
+
+You can also use the SOS Debugging Extension by loading it into the [WinDbg/dbg debugger](/windows-hardware/drivers/debugger/debugger-download-tools) and executing commands within the Windows debugger.  SOS commands can be used on live processes or dumps.
+
+To load the SOS Debugging Extension into the Windows debugger, run the following command in the tool:
+
+```console
+.loadby sos clr
+```
+
+WinDbg.exe and Visual Studio use a version of SOS.dll that corresponds to the version of Mscorwks.dll currently in use. By default, you should use the version of SOS.dll that matches the current version of Mscorwks.dll.
+
+To use a dump file created on another computer, make sure that the Mscorwks.dll file that came with that installation is in your symbol path, and load the corresponding version of SOS.dll.
+
+To load a specific version of SOS.dll, type the following command into the Windows Debugger:
+
+```console
+.load <full path to sos.dll>
+```
+
+### LLDB Debugger
+
+Please see [dotnet-sos](dotnet-sos) for instructions on configuring SOS for LLDB. SOS commands can be used on live processes or dumps.<br/><br/>
+By default you can reach all the SOS commands by using: _sos [command\_name]_
+However the common commands have been aliased so that you don't need the SOS prefix:
+
+| Command                             | Function                                                                                      |
+| ----------------------------------- | --------------------------------------------------------------------------------------------- |
+|    bpmd                             | Creates a breakpoint at the specified managed method in the specified module.
+|    clrstack                         | Provides a stack trace of managed code only.
+|    clrthreads                       | List the managed threads running.
+|    clru                             | Displays an annotated disassembly of a managed method.
+|    dso                              | Displays all managed objects found within the bounds of the current stack.
+|    dumpasync                        | Displays info about async state machines on the garbage-collected heap.
+|    dumpclass                        | Displays information about a EE class structure at the specified address.
```

Comment:
```suggestion
|    dumpclass                        | Displays information about the EE class structure at the specified address.
```

Will it be obvious to readers what EE is?

Reply:
EEClass is an internal data structure.  The "Dump*" SOS commands display internal .NET Runtime data structures in the debugger.  I removed space between "EE" and "class".  Please let me know if there's anything else I should do to further clarify it.